src/csharp/Grpc.Core/Internal/NativeMethods.Generated.cs

```diff
@@ -478,7 +478,7 @@ public class Delegates
             public delegate void grpcsharp_channel_args_set_integer_delegate(ChannelArgsSafeHandle args, UIntPtr index, string key, int value);
             public delegate void grpcsharp_channel_args_destroy_delegate(IntPtr args);
             public delegate void grpcsharp_override_default_ssl_roots_delegate(string pemRootCerts);
-            public delegate ChannelCredentialsSafeHandle grpcsharp_ssl_credentials_create_delegate(string pemRootCerts, string keyCertPairCertChain, string keyCertPairPrivateKey);
```

Comment:
this file is generated - you need to edit the template and run `tools/buildgen/generate_projects.sh`

Reply:
I ran the script, it generated quite many changes but I assumed that only the one for this files were relevant.